### DISCUSSION TITLE
clickbench 单表压测问题

### DISCUSSION THREAD

**Discussion by leonadle** [ByConity/ByConity#377] [Q&A] [answered]
## 问题
clickhouse(22.9.3) 和 byconity(0.1.0) 单表压测对比后发现大部分查询都是略慢于 clickhouse，这种情况正常吗，是否有更加官方的压测报告？

**Comment by hustnn** [comment, marked as answer]
正常的，ByConity有一定额外的开销，我们也可以内部再测一个版本。你测的CH是基于分布式表还是local表？

**Comment by leonadle** [comment]
> 正常的，ByConity有一定额外的开销，我们也可以内部再测一个版本。你测的CH是基于分布式表还是local表？
测的是local表

**Comment by hustnn** [comment]
贴一个下结果？最好基于分布式表来对比

**Comment by leonadle** [comment]
> 贴一个下结果？最好基于分布式表来对比
分布式也测了，大部分查询还是稍慢的。都是用 docker-compose 部署测试的
<img width="829" alt="image" src="https://github.com/ByConity/ByConity/assets/58548234/f62fc663-270a-427f-8467-7c0ceef18cf9">

**Reply by serenallttyy** [reply]
hello，看到您是在关注 ByConity 社区，可以加入我们的社群交流。
<img width="117" alt="小助手二维码（新）" src="https://github.com/ByConity/ByConity/assets/88301094/75fb0cfc-5c2e-470f-8ce0-19602eb14722">

**Comment by hustnn** [comment]
@leonadle 好，上传一下完整的结果？谢谢，我们也会内部复现一下
